loop_wcbench.sh

```diff
@@ -9,6 +9,9 @@ EX_OK=0
 # Output verbose debug info (true) or not (anything else)
 VERBOSE=false
 
+# Initialisation for looping N times
+n=1
```

Comment:
We likely don't need a global to initialize a loop. Might need a global default if we make num_runs optional.